spec/index.bs

```diff
@@ -7014,6 +6997,114 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
         or the third item of the sequence.
 </div>
 
+# Feature Definitions # {#feature-definitions}
+
+The capabilities of each [=feature=] approved by the WebGPU Working Group are defined here.
+
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>depth-clamping</dfn> ## {#depth-clamping}
+
+Issue: Define functionality when the {{GPUFeatureName/"depth-clamping"}} [=feature=] is enabled.
+
+**Feature Dictionary Values**
```

Comment:
nit: `###`? Or maybe these mini-headers aren't really necessary?

Reply:
I didn't want to make this into a proper header because then it would show up in the sidebar and look rather cluttered. The "pseudo headers" felt like they'd be helpful if we have more complex features someday that define some combination of new enums, dictionary entries, or functions. I can remove them if you'd prefer, though.

Reply:
I would like if there were a way we could make a header that doesn't show up in the ToC. I believe if you go all the way down to h4 or h5 then it doesn't show up in the ToC. But I don't know if it's valid to nest an h5 in an h2.